File: PythonGtu/gtu/_test/test001.py
import sys
# sys.path.append("D:\workstuff\gtu-test-code\PythonGtu")

from enum import Enum
from gtu.reflect import checkSelf
import re


class Test001Class :
    def __init__(self):
        print("Test001Class Done!!!")


def main():
    ptn = re.compile("www\.youtube\.com\/watch\?v\=(\w+?)\&")
#     mth = ptn.search("https://www.youtube.com/watch?v=S02GWKdZpbg&t=10486s")
    mth = ptn.search("https://www.youtube.com/watch?v=S02GWKdZpbg")
    if mth is not None :
        print("group1 = ", mth.group(1))
    else :
        print("____")



if __name__ == '__main__' :
    main()
    print("done...")

File: GTU/node_js/util/test_001.js
const {RootDirHolder} = require("D:/workstuff/gtu-test-code/GTU/node_js/system/rootDirHolder.js");
new RootDirHolder();

const checkSelf = require(rootDirHolder.path("/util/checkSelf"));
const jsonUtil = require(rootDirHolder.path("/util/jsonUtil"));
const numberUtil = require(rootDirHolder.path("/util/numberUtil"));

const fs = require("fs");

console.log("<<<" + typeof(fs) + " , " + fs);
        console.log("----------------------------------------start");
        for(let i in fs){
            console.log("---" + i);
        }
        console.log("----------------------------------------end");